Vagrantfile

```diff
@@ -0,0 +1,20 @@
+Vagrant.configure('2') do |config|
+  config.ssh.forward_agent = true
+  config.vm.box = 'cargomedia/debian-7-amd64-default'
+
+  config.vm.hostname = 'janus-gateway-js.dev.cargomedia.ch'
+
+  config.vm.network :private_network, ip: '10.10.10.16'
```

Comment:
Since you won't be using any connection to the box besides `vagrant ssh` you can skip line 7 entirely